libs/zems/zems.zig

```diff
@@ -0,0 +1,119 @@
+pub const is_emscripten = true;
```

Comment:
If user is linking zems; either directly or by enabling emsc support in zglfw then we should be able to detect that without this